docs/developer-guide/base-maps/using-with-maplibre.md

```diff
@@ -0,0 +1,248 @@
+# Using with Maplibre
+
+| Pure JS | React | Overlaid | Interleaved |
+| ----- | ----- | ----- | ----- |
+|  ✓ | ✓ | [example](https://github.com/visgl/deck.gl/tree/master/examples/get-started/pure-js/maplibre) | [example](https://deck.gl/gallery/maplibre-overlay) |
+
+[Maplibre GL JS](https://maplibre.org/maplibre-gl-js/docs/) is a powerful open-source library that uses WebGL to render interactive maps from vector tiles in a browser. deck.gl's `MapView` can sync perfectly with the camera of Maplibre, at every zoom level and rotation angle.
+
+## Integration Modes
+
+When using deck.gl and Maplibre, there are three options you can choose from: interleaved, overlaid, and reverse-controlled.
+
+### Interleaved
+
+The [interleaved](../../get-started/using-with-map.md#interleaved) mode renders deck.gl layers into the WebGL2 context created by Maplibre. If you need to mix deck.gl layers with Maplibre layers, e.g. having deck.gl surfaces below text labels, or objects occluding each other correctly in 3D, then you have to use this option.
+
+Interleaving is supported by using [MapboxOverlay](../../api-reference/mapbox/mapbox-overlay.md) with `interleaved: true`. It requires WebGL2 and therefore only works with `maplibre-gl@>3`. See [compatibility](../../api-reference/mapbox/overview#interleaved-renderer-compatibility) and [limitations](../../api-reference/mapbox/overview.md#limitations).
```

Comment:
You know what I think about the name "mapbox" in both the npm package and the class name so I won't repeat it here.